plugins/catalog-backend-module-github/package.json

```diff
@@ -44,6 +45,8 @@
     "@backstage/plugin-catalog-node": "workspace:^",
     "@backstage/types": "workspace:^",
     "@octokit/graphql": "^5.0.0",
+    "@octokit/rest": "^19.0.4",
```

Comment:
Needed to add the octokit/rest library because the graphql does not support code search